lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -2896,6 +2896,44 @@ class DecomposeAtenMseLossOp : public OpRewritePattern<AtenMseLossOp> {
 };
 } // namespace
 
+namespace {
+// Decompose `valsem.aten.fill.Tensor` op into `valsem.aten.fill.Scalar` op.
+class DecomposeValsemVariantAtenFillTensorOp
```

Comment:
It's much simpler to add an elementwise pattern for this op. See the implementation of `aten.fill.Scalar` here: https://github.com/llvm/torch-mlir/pull/1531